allensdk/brain_observatory/sync_dataset.py

```diff
@@ -104,20 +110,25 @@ class Dataset(object):
                        "cam1_exposure",
                        "behavior_monitoring"}
 
-    def __init__(self, path):
+    def __init__(self, path, sync_line_label_deprecation_warning=False):
+        '''
+        sync_line_label_deprecation_warning: (bool) if True,
+        warns about deprecated lines in sync file
+        '''
         self.dfile = self.load(path)
-        self._check_line_labels()
+        self._check_line_labels(
```

Comment:
In light of:
https://github.com/AllenInstitute/AllenSDK/pull/2074#issuecomment-811378281

Maybe just remove this line and the `_check_line_labels()` method entirely. The original intention was to let the analyst know that the sync file from that 'era' is not particularly trustworthy. Given that I also dimly recall that the line swapping process itself was pretty rocky, I still stand by the assertion that those sync files are not terribly trustworthy.

But if folks are not doing anything actionable with the warnings, then maybe it doesn't make sense to have them.